rpc-client-api/src/response.rs

```diff
@@ -487,6 +487,9 @@ pub struct RpcConfirmedTransactionStatusWithSignature {
 pub struct RpcPerfSample {
     pub slot: Slot,
     pub num_transactions: u64,
+    /// `num_non_vote_transactions` is only reported starting from version 1.15.X.
+    // TODO Make sure the version number is correct.
```

Comment:
In the same vein as Steve's comments on the previous PR, I'm inclined to say remove this, and use the commit history if we need to know which version added this field.

Reply:
I thought this is something that the API consumers can see.
In which case, it seems useful?
Also, it makes it clear that it is not just something that might be absent, but instead, the field is always present starting with v1.15 onwards.

And the last argument is that in the previous PR, it was about a version where backwards compatibility should be removed.  And that is indeed tricky.
While here, it just states that this field is now always present.

I removed the `TODO` line.

Obviously, I am not going to document every line with a comment of when it was added :)
But for APIs, versioning seems like an almost must-have, for users to understand what they can rely on.

What do you think?

Reply:
I still think not documenting the nature of the `Option` wrapper is bad.
But I have removed the comment.